catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliManagerType.java

```diff
@@ -44,10 +44,6 @@ public String getSpecName() {
         return specName;
     }
 
-    public void setSpecName(String specName) {
-        this.specName = specName;
-    }
```

Comment:
✏️ Can specName be `final` now that this method is removed?

Reply:
@emmberk  specName is being set in the constructor.

Reply:
Right. `specName` should only ever be set once in the constructor.